packages/ember-metal/lib/property_get.js

```diff
@@ -76,7 +76,7 @@ export function _getPath(root, path) {
 
   for (let i = 0; i < parts.length; i++) {
     if (obj == null) {
```

Comment:
we only want to call get on object like things, this should do if (!isObjectLike(obj)) { return undefined; }